core/src/main/scala/org/apache/spark/status/AppStatusStore.scala

```diff
@@ -421,6 +420,26 @@ private[spark] class AppStatusStore(
     constructTaskDataList(taskDataWrapperIter)
   }
 
+  def exceptionSummary(stageId: Int, attemptId: Int): Seq[v1.ExceptionSummary] = {
+    val tasks = taskList(stageId, attemptId, Int.MaxValue)
+    tasks.filter(t => t.status.equalsIgnoreCase("failed"))
+      .flatMap(t => t.errorMessage)
```

Comment:
If an exception doesn't have any error message, it can't appear in the summary.

Reply:
I used flatMap here, if errorMessage is `None`, it won't appear in the summary. Or do you mean empty string?

Reply:
Sometimes exceptions can have no error message.
For example, the summary is not shown with the following example.
```
sc.parallelize(1 to 10).foreach(_ => throw new RuntimeException())
```

Reply:
As the group by condition is exceptionType and message, so I just group them together. Or do you think I should filter them out with `filter(!errorMessage.isEmpty)`?

Reply:
I wonder can we show the count somehow even though exceptions have no error message.